test/e2e/logforwarding/fluent/fluent_test.go

```diff
@@ -73,10 +78,11 @@ var _ = Describe("[ClusterLogForwarder]", func() {
 				})
 			}
 			f.Create(c.Client)
-			for _, name := range []string{"application", "infrastructure", "audit"} {
+			time.Sleep(30 * time.Second)
```

Comment:
can we use polling instead of sleep

Reply:
I would but I don't know what to poll for. This sleep greatly lessens the chance of a rather frequent error where we pick up infra logs **without**  the "log-type":"infrastructure" label if we look into what the receiver receives "too soon". This needs more investigation.

```
• Failure [18.107 seconds]
[ClusterLogForwarder]
/home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_test.go:17
  with app/infra/audit receiver
  /home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_test.go:28
    forwards different types to different outputs with labels [It]
    /home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_test.go:64

    Expected
        <string>: {"_STREAM_ID":"f6f5e3f227e24e7c953524465618bd50","_SYSTEMD_INVOCATION_ID":"0ad2843bceb54d2196247d08c20725aa","systemd":{"t":{"BOOT_ID":"79f4d056568c4f1c81d811f2edf6c24e","CAP_EFFECTIVE":"3fffffffff","CMDLINE":"kubelet --config=/etc/kubernetes/kubelet.conf --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig --kubeconfig=/var/lib/kubelet/kubeconfig --container-runtime=remote --container-runtime-endpoint=/var/run/crio/crio.sock --runtime-cgroups=/system.slice/crio.service --node-labels=node-role.kubernetes.io/worker,node.openshift.io/os_id=rhcos --node-ip= --minimum-container-ttl-duration=6m0s --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec --cloud-provider=aws --pod-infra-container-image=quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:5994ce07b8fa5b3fa9a1ecd99f509ccf960918d9520d60fdf0087d0d656d6b23 --v=2","COMM":"kubelet","EXE":"/usr/bin/kubelet","GID":"0","MACHINE_ID":"ec23533db1b84b76e7115926d5c40aac","PID":"1655","SELINUX_CONTEXT":"system_u:system_r:unconfined_service_t:s0","STREAM_ID":"f6f5e3f227e24e7c953524465618bd50","SYSTEMD_CGROUP":"/system.slice/kubelet.service","SYSTEMD_INVOCATION_ID":"0ad2843bceb54d2196247d08c20725aa","SYSTEMD_SLICE":"system.slice","SYSTEMD_UNIT":"kubelet.service","TRANSPORT":"stdout","UID":"0"},"u":{"SYSLOG_FACILITY":"3","SYSLOG_IDENTIFIER":"hyperkube"}},"level":"info","message":"I0405 21:07:29.094435    1655 reflector.go:225] Stopping reflector *v1.Secret (0s) from object-\"170029-works-when-multiple-outputs-use-same-secret-05a9d37f\"/\"default-dockercfg-94pn5\"","hostname":"ip-10-0-164-107","pipeline_metadata":{"collector":{"ipaddr4":"10.0.164.107","inputname":"fluent-plugin-systemd","name":"fluentd","received_at":"2021-04-05T21:12:47.084086+00:00","version":"1.7.4 1.6.0"}},"@timestamp":"2021-04-05T21:07:29.094465+00:00","viaq_index_name":"infra-write","viaq_msg_id":"Zjk4NjIyM2ItMTVlOS00NmE3LWFjYjUtMGNlOTZmNjQzZDUy","kubernetes":{}}
        
    To satisfy at least one of these matchers: [%!s(*matchers.EqualMatcher=&{}) %!s(*matchers.ContainSubstringMatcher=&{"log-type":"infrastructure" []})]

    /home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_test.go:83

    Full Stack Trace
    github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent_test.glob..func2.3.4()
    	/home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_test.go:83 +0x885
    github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent_test.TestFluent(0xc000682780)
    	/home/syedriko/go/src/github.com/openshift/cluster-logging-operator/test/e2e/logforwarding/fluent/fluent_suite_test.go:12 +0x98
    testing.tRunner(0xc000682780, 0x15405c8)
    	/usr/local/go/src/testing/testing.go:1194 +0xef
    created by testing.(*T).Run
    	/usr/local/go/src/testing/testing.go:1239 +0x2b3
------------------------------

```